SU-SWS/stanford_basic
# outline instead of border

This is a suggestion to test out.

# READY FOR REVIEW 

# Summary
- Perhaps try out `outline` instead of border on the `page-content` region as that may help prevent the content shift that happens on hover.

# Review By (Date)
- meh

# Urgency
- low

# Steps to Test

1. Check out this branch
2. Compile the css (npm run publish) 
3. Clear cache and try it out on a page
